test/ParallelIOTest.cpp

```diff
@@ -287,6 +287,58 @@ TEST_CASE( "no_parallel_hdf5", "[parallel][hdf5]" )
 
 #endif
 
+// this one works for both ADIOS1 and ADIOS2
+#if (openPMD_HAVE_ADIOS1 || openPMD_HAVE_ADIOS2) && openPMD_HAVE_MPI
+void
+available_chunks_test( std::string file_ending )
+{
+    int __mpi_rank{ -1 }, __mpi_size{ -1 };
+    MPI_Comm_rank( MPI_COMM_WORLD, &__mpi_rank );
+    MPI_Comm_size( MPI_COMM_WORLD, &__mpi_size );
+    unsigned mpi_rank{ static_cast< unsigned >( __mpi_rank ) },
+        mpi_size{ static_cast< unsigned >( __mpi_size ) };
```

Comment:
Slipped through: reserved naming.

Idea (r for read or so) - did not check all the member vars and hope it does not shadow anything this way:
```suggestion
    int r_mpi_rank{ -1 }, r_mpi_size{ -1 };
    MPI_Comm_rank( MPI_COMM_WORLD, &r_mpi_rank );
    MPI_Comm_size( MPI_COMM_WORLD, &r_mpi_size );
    unsigned mpi_rank{ static_cast< unsigned >( r_mpi_rank ) },
        mpi_size{ static_cast< unsigned >(r_mpi_size ) };
```

Reply:
Yeah, that works